### PR TITLE
Verify ephemeron key is not immediate when marking

### PR DESCRIPTION
**Pull request by guillep**
Ephemerons with immediate keys should not fail

### CODE REVIEW

**Comment by guillep** [comment]
CI is green, timed out in windows...

File: smalltalksrc/VMMakerTests/VMSpurOldSpaceGarbageCollectorTest.class.st [hunk 1]
```diff
@@ -416,6 +416,24 @@ VMSpurOldSpaceGarbageCollectorTest >> testEphemeronOverflowUnscannedEphemeronQue
...
+	memory storePointer: 0 ofObject: ephemeron1 withValue: key.
+
+	memory setCheckForLeaks: 63. "all"
+	memory fullGC.
+
+	self
+		assert:
+		(memory fetchInteger: 0 ofObject: self keptObjectInVMVariable1)
+		equals: 15
+]
```
Comment (PalumboN) [inline smalltalksrc/VMMakerTests/VMSpurOldSpaceGarbageCollectorTest.class.st:435]: Why here the assertion says just `15` instead of `key`? @guillep

Comment (guillep) [reply smalltalksrc/VMMakerTests/VMSpurOldSpaceGarbageCollectorTest.class.st:435]: I did a `fetchInteger:`, to make sure the same value is there.
What would be the most readable?
I think in any case we will need a comment, right?

Comment (PalumboN) [reply smalltalksrc/VMMakerTests/VMSpurOldSpaceGarbageCollectorTest.class.st:435]: Ahhhh okkkkk. I don't think that a comment is needed, I just need to learn how to read 😅 

Maybe having `storeInteger:` also would help with the _symmetry_.

**Pull request merged**